diff --git a/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx b/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx
index 73e12af..13d0ded 100644
--- a/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx
+++ b/node_modules/react-native-radio-buttons-group/lib/RadioButton.tsx
@@ -6,8 +6,8 @@ import { RadioButtonProps } from './types';
 export default function RadioButton({
   accessibilityLabel,
   borderColor,
-  borderSize = 2,
-  color = '#444',
+  borderSize = 1,
+  color = '#274e8c',
   containerStyle,
   description,
   descriptionStyle,
@@ -48,6 +48,7 @@ export default function RadioButton({
         onPress={handlePress}
         style={[
           styles.container,
+          selected && {backgroundColor: '#dce6f5'},
           orientation,
           { opacity: disabled ? 0.2 : 1 },
           containerStyle,
@@ -59,7 +60,7 @@ export default function RadioButton({
           style={[
             styles.border,
             {
-              borderColor: borderColor || color,
+              borderColor: selected ? (borderColor || color) : '#c2c2c2',
               borderWidth,
               width: sizeFull,
               height: sizeFull,
@@ -87,8 +88,10 @@ export default function RadioButton({
 const styles = StyleSheet.create({
   container: {
     alignItems: 'center',
-    marginHorizontal: 10,
     marginVertical: 5,
+    width: '100%',
+    padding: 10,
+    borderRadius: 8
   },
   border: {
     justifyContent: 'center',
diff --git a/node_modules/react-native-radio-buttons-group/lib/RadioGroup.tsx b/node_modules/react-native-radio-buttons-group/lib/RadioGroup.tsx
index 328b43c..d99c252 100644
--- a/node_modules/react-native-radio-buttons-group/lib/RadioGroup.tsx
+++ b/node_modules/react-native-radio-buttons-group/lib/RadioGroup.tsx
@@ -6,9 +6,9 @@ import { RadioGroupProps } from './types';
 
 export default function RadioGroup({ containerStyle, layout = 'column', onPress, radioButtons, selectedId, testID}: RadioGroupProps) {
 
-  function handlePress(id: string) {
+  function handlePress(button: any, id: string) {
     if(id !== selectedId && onPress) {
-      onPress(id);
+      onPress(button);
     }
   }
 
@@ -19,7 +19,7 @@ export default function RadioGroup({ containerStyle, layout = 'column', onPress,
           {...button}
           key={button.id}
           selected={button.id === selectedId}
-          onPress={() => handlePress(button.id)}
+          onPress={() => handlePress(button, button.id)}
         />
       ))}
     </View>
